.github/README.md

```diff
@@ -1,164 +1,74 @@
-
-
-## How to Thank us
-
-Being an open-source project, we rely on volunteers to pursue development. Here are ways to help us if you use AzerothCore:
-
-### Github Star
-
-Click on the "star this repository" button to help us gain more visibility on Github!
-
-### By contributing
-
-Check the **CONTRIBUTING** section below.
-
-### Financially :moneybag:
-
-You can support the AzerothCore by [donating](https://www.paypal.com/donate/?hosted_button_id=L69ANPSR8BJDU).
-The money will be used to pay freelance developers for more open-source fixes.
-
-### Advertising
-
-By talking about us on different platforms or to people who would like to get involved.
-
-
-## Contributing
-
-AzerothCore is a learning project, and there are lots of different ways to contribute to the project:
-
-* By [testing our fixes](http://www.azerothcore.org/wiki/How-to-test-a-PR) (we can teach you how to correctly use Git to help us but that will also help you out tremendously)
-* By developing directly to the core or the modules
-* By reporting bugs within the project
-* By [creating new modules](http://www.azerothcore.org/wiki/Create-a-Module)
-* By improving our wiki
-* By providing direct support to our community (on Discord, StackOverflow or specialized forums)
-* By making extra content (video tutorial for example)
-* By putting bounties on issues
+* Blizzlike content
+  * We strive to make all in-game content to be blizzlike. Therefore we have a high standard for fixes being made.
 
+* Customization
+  * It should be easy to customize your experience using [modules](#modules).
 
-If you want to contribute to the project, you will find a lot of resources that will guide you in our wiki.
+* Community driven
+  * AzerothCore has an active community of developers, contributors, and users who collaborate, share knowledge, and provide support through forums, Discord channels, and other communication platforms 
 
-Feel free to join us on [our Discord chat server](https://discord.gg/gkt4y2x) where we teach a lot of new people how to get started and who are now important contributors!
+### Modules
 
-<!-- TO UNCOMMENT LATER -->
-<!-- As we put a big emphasis on community, there are also special rewards for contributors such as reputation ranks (displayed in our Discord and our website), reputation badges (to display on your project/portfolio), premium software licenses, private modules access, private tools access and a lot of small private repositories access. -->
+AzerothCore is designed to be highly modular, allowing developers to extend and customize the game to suit their preferences or create unique gameplay experiences. This flexibility enables the addition of custom features, content, and modifications.
 
+We have a lot of modules already made by the community which can be found in the [Module Catalouge](https://www.azerothcore.org/catalogue.html#/).
```

Comment:
```suggestion
We have a lot of modules already made by the community, many of which can be found in the [Module Catalogue](https://www.azerothcore.org/catalogue.html#/).
```

Not all modules are displayed in the catalogue, both due to an issue (iirc) and not all modules being labelled appropriately.